README.md

```diff
@@ -6,7 +6,7 @@ Lightweight server to translate Grafana webhook to Feishu card.
 
 The program needs two environment variables:
 
-- `FEISHU_WEBHOOK`: The web hook URL to push Feishu notifications. Should be something like `https://open.feishu.cn/open-apis/bot/v2/hook/a21843-123-123-abc987`
+- `FEISHU_WEBHOOK_BASE`: (Optional) The web hook base URL to push Feishu notifications. Default is `https://open.feishu.cn/open-apis/bot/v2/hook`
```

Comment:
Well, I don't think there is a need to configure `FEISHU_WEBHOOK_BASE`. What else can it be?

We can remove `FEISHU_WEBHOOK_BASE` and only document `FEISHU_WEBHOOK` environment variable.

```suggestion
- `FEISHU_WEBHOOK`: (Optional) The web hook URL to push Feishu notifications. Should be something like `https://open.feishu.cn/open-apis/bot/v2/hook/a21843-123-123-abc987`. If it is not provided, the bot UUID should be provided in the path. See below for detail.
```

Reply:
I thought there maybe corner cases where people might want to define there own API base. I don't know if lark suite (the oversea version) has the same bot functionalities but with 'larksuite.com' domain name. Or feishu might decide to roll out v3 API instead of v2. 

To minimize burden on users, we can remove the document. But maybe we want to keep the code configurable, as well as the prebuit container images.

Anyway, you decide this.